### PR TITLE
Fixes on links & Added GIFs on correct implementations

### PR DESCRIPTION
**Pull request by mateosauton**
Corrected hyperlink issues and introduce GIFs demonstrating the proper use of 'Verify' and 'Pay' commands.

### CODE REVIEW

File: src/pages/mini-apps/commands/pay.mdx [hunk 1]
```diff
@@ -266,3 +266,13 @@ export async function POST(req: NextRequest) {
 	}
 }
 ```
+## Success Result on World App
+
+If implemented correctly, the user will see the following drawer on World App.
+
+    <div className='grid justify-items-center text-center'>
+        <video className="m-auto" width="300" autoPlay muted loop playsInline>
+            <source src="/images/docs/mini-apps/commands/PayCommand.mp4" type="video/mp4" />
```
Comment (michalstruck) [inline src/pages/mini-apps/commands/pay.mdx:275]: keep it uniform with the other .mp4 naming convention

File: src/pages/mini-apps/commands/verify.mdx [hunk 2]
```diff
@@ -201,3 +201,14 @@ export async function POST(req: NextRequest) {
 	}
 }
 ```
+## Success Result on World App
+
+If implemented correctly, the user will see the following drawer on World App.
+
+<div className='grid justify-items-center text-center'>
+    <video className="m-auto" width="300" autoPlay muted loop playsInline>
+        <source src="/images/docs/mini-apps/commands/Verify Command.mp4" type="video/mp4" />
```
[the remark this answers is not in the document]
Comment (mateosauton) [reply src/pages/mini-apps/commands/verify.mdx]: done

File: src/pages/mini-apps/commands/pay.mdx (hunk 1 above)
Comment (mateosauton) [reply src/pages/mini-apps/commands/pay.mdx]: done

File: src/pages/mini-apps/quick-start/app-store.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Link } from '@/components/Link'
 
 # App Store
-To ensure a consistent user experience, apps will be rejected unless they follow our <Link href="/mini-apps/more/guidelines">guidelines</Link>.
+To ensure a consistent user experience, apps will be rejected unless they follow our <Link href="/mini-apps/design/app-guidelines">guidelines</Link>.
```
Comment (michalstruck) [inline src/pages/mini-apps/quick-start/app-store.mdx:4]: this route is non-existent

**Review by michalstruck — APPROVED** [review]

**Pull request merged**